tests/test_io/test_web/test_load.py

```diff
@@ -1,68 +1,116 @@
+"""Test model loading from local and remote model repositores."""
+
 import gzip
-import pathlib
+from pathlib import Path
+from typing import TYPE_CHECKING, Callable
+from unittest.mock import Mock
 
 import pytest
 
 from cobra import Configuration
 from cobra.io import BiGGModels, BioModels, load_model
 
 
+if TYPE_CHECKING:
+    from pytest_mock import MockerFixture
+
+    from cobra import Model
+
+
 @pytest.fixture(scope="module")
-def mini_sbml(data_directory):
+def mini_sbml(data_directory: Path) -> bytes:
     """Provide a gzip-compressed SBML document."""
-    with (pathlib.Path(data_directory) / "mini_cobra.xml").open(mode="rb") as handle:
+    with open(data_directory / "mini_cobra.xml", "rb") as handle:
```

Comment:
```suggestion
    with (data_directory / "mini_cobra.xml").open(mode="rb") as handle:
```

Reply:
Any specific reason for using `Path.open()`?

Reply:
Okay! 😆